src/connectivity.jl

```diff
@@ -123,7 +133,7 @@ function discover_vertex!(vis::TarjanVisitor, v)
 end
 
 function examine_neighbor!(vis::TarjanVisitor, v, w, w_color::Int, e_color::Int)
-    if w_color > 0 # 1 means added seen, but not explored; 2 means closed
+    if w_color >= 0 # >=0 means added seen
```

Comment:
Careful. See https://github.com/JuliaGraphs/LightGraphs.jl/issues/71, https://github.com/JuliaLang/Graphs.jl/issues/187, and https://github.com/JuliaLang/Graphs.jl/pull/188.